sda-commons-server-mongo-testing/src/test/java/org/sdase/commons/server/mongo/testing/UseExistingMongoDbClassExtensionTest.java

```diff
@@ -41,8 +41,10 @@ public void initUseExistingMongoDbExtension() {
             + "?"
             + EXTERNAL_DB.getOptions();
     originalMongoDbConnectionString =
-        System.getenv().get(OVERRIDE_MONGODB_CONNECTION_STRING_ENV_NAME);
-    Environment.setEnv(OVERRIDE_MONGODB_CONNECTION_STRING_ENV_NAME, MONGODB_CONNECTION_STRING);
+        new SystemPropertyAndEnvironmentLookup()
+            .lookup(OVERRIDE_MONGODB_CONNECTION_STRING_SYSTEM_PROPERTY_NAME);
+    System.setProperty(
+        OVERRIDE_MONGODB_CONNECTION_STRING_SYSTEM_PROPERTY_NAME, MONGODB_CONNECTION_STRING);
```

Comment:
Do we need to save and reset?

Reply:
Test test setup is a little strange but the value is reset manually in `AfterEach`. I would rather not touch the code in this PR.

Reply:
Ups, sorry for the noise. I did not see that the old value is restored.